frameworks/JavaScript/nodejs/handlers/mysql-raw.js

```diff
@@ -21,6 +26,16 @@ const queries = {
   }
 };
 
+const populateCache = (callback) => {
+  if (cachePopulated) return callback();
+  connection.query(queries.ALL_WORLDS, (err, rows) => {
```

Comment:
Will the CachedWorld table be used (as the specification says right now), or the regular World table?